core/ui/vault/deposit/DepositForm/ActionSpecific/UnmergeSpecific/UnmergeSpecific.tsx

```
@@ -0,0 +1,44 @@
+import { useCoreViewState } from '../../../../../navigation/hooks/useCoreViewState'
+import { useCurrentVaultCoins } from '../../../../state/currentVaultCoins'
+import { useRujiBalanceQuery } from '../../../hooks/useRujiBalanceQuery'
+import { useDepositFormHandlers } from '../../../providers/DepositFormHandlersProvider'
+import { InputFieldWrapper } from '../../DepositForm.styled'
+
+export const UnmergeSpecific = () => {
+  const [{ register }] = useDepositFormHandlers()
+  const [{ coin: coinKey }] = useCoreViewState<'deposit'>()
+  const coin = getCoinFromCoinKey(coinKey)
+  const coinAddress = useCurrentVaultCoins().find(
+    coin => coin.id === coin.id
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)